pkg/ipam/allocator_test.go

```diff
@@ -20,9 +20,11 @@ import (
 
 type ownerMock struct{}
 
-func (o *ownerMock) K8sEventReceived(scope string, action string, valid, equal bool) {}
-func (o *ownerMock) K8sEventProcessed(scope string, action string, status bool)      {}
-func (o *ownerMock) RegisterCiliumNodeSubscriber(s subscriber.CiliumNode)            {}
+func (o *ownerMock) K8sEventReceived(resourceApiGroup, scope string, action string, valid, equal bool) {
+}
+func (o *ownerMock) K8sEventProcessed(scope string, action string, status bool) {
+}
```

Comment:
Nit:

```suggestion
func (o *ownerMock) K8sEventReceived(resourceApiGroup, scope string, action string, valid, equal bool) {}
func (o *ownerMock) K8sEventProcessed(scope string, action string, status bool) {}
```

Reply:
go fmt actually puts the parenthesis of K8sEventReceived on a newline, I can only assume because of the line length.